concepts/observability-strategy/configurable-tracing/03-tracing-pocs.md

```diff
@@ -32,7 +32,26 @@ Comparing the resource consumption and throughput of the envoys, and checking ot
 
 ## Tail Sampling
 
-Understand what tail sampling strategies are available already in OTEL and understand the scaling requirements of them.
+Opposed to Head-based sampling, Tail sampling make the decision at the end of the entire flow, wenn whole trace data already gathered. This kind of sampling decision made at the collector level.
+
+With Tail sampling, it's possible to create advanced rules to filter out traces based on any **span** property, include their attributes, duration etc. Tail sampling will allow us to collect data like unsually long operations and rare errors.
+
+Basically, with Tail sampling, sampling decision delayed end of flow untill all spans of a trace are available, this enables better sampling decisions based on all data from the trace.
+
+However, make decision at the end of the trace, the backed has to buffer entire trace data, which can increase storage overhead.
+
+Choosing right sample rate is diffucult, mostly depend on system requirement, the way services are built and amount of traffic they have. For example when a service is verry noisy and receiving a lot of traffic, rather a small percentece of sampling rate is a better decision, to avoid the costs and the noise. But when there is a endpoint with less traffic (e.g. a core service), high percentece is a better chooice since it won't cost much but most likely be valuable.
```

Comment:
```suggestion
Choosing the right sample rate is difficult - it mostly depends on system requirements, the way services are built,  and the amount of traffic they have. For example, when a service is very noisy and receives a lot of traffic, a smaller percentage of sampling rate is better, to prevent the costs and the noise. But for an endpoint with less traffic (like a core service), high percentage is a better choice, because it it won't cost much and, most likely, is valuable.
```